base.js

```diff
@@ -1,125 +1,45 @@
 module.exports = {
-  extends: 'eslint:recommended',
+  extends: 'airbnb-base',
   parser: 'babel-eslint',
   env: {
-    es6: true
+    es6: true,
   },
   parserOptions: {
-    ecmaVersion: 6
+    ecmaVersion: 6,
```

Comment:
aww i understand this `,` will make diffs easier to read but i personally think it looks a bit ugly haha. :woman_shrugging: